_posts/2022-03-07-gsoc2022.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: Google Summer of Code 2022
+---
+
+<p>
+<img
+src="https://developers.google.com/open-source/gsoc/resources/downloads/GSoC-Vertical.svg"
+title="Google Summer of Code 2022" alt="Google Summer of Code with
+MDAnalysis 2022"
+style="float: right; height: 10em; " />
+</p>
+
+MDAnalysis has been accepted as an [organization][org] into [Google Summer of Code 2022][gsoc]!
+If you are a student who is interested in working with us
+this summer, please read the advice and links below and write to us on the
+[mailing list].
+
+We are looking forward to all applications. Note that the GSoC program was
+[updated](https://opensource.googleblog.com/2021/11/expanding-google-summer-of-code-in-2022.html)
+compared to previous years: GSoC will welcome not just students, but *any new
+and beginner open source contributors over 18 years old*. Projects are also now
+scoped as either 175-hour (medium) or 350-hour (long) size. Finally, the duration
+can be extended from the standard 12 weeks to 22 weeks.
+
+The application window **deadline** is **April 19, 2022 18:00 (UTC)**. As
+part of the application process you must familiarize yourself with [Google
+Summer of Code 2022][gsoc]. 
+
+If you are interested in working with us please read on and contact us on our
+[mailing list].  [Apply] as soon as possible; the application window opens on
+April 4, 2022 but potential GSoC contributors are expected to familiarize
+themselves with application requirements and mentoring organizations as soon as
+possible. It's also never too early to discuss application ideas with us!
+
+# Project Ideas
+
+If you have your own idea about a potential project we'd love to work with you
+to develop this idea; please write to us on the [developer list][mailing list]
+to discuss it there.
+
+We also have listed several [possible projects][ideas] for you to work on. Our
+initial list of ideas (see summaries in the table below) contains various
+projects of different scope and with different skill requirements. However,
+check the [ideas][] page — we might add more ideas after the posting date of
+this post.
+
+You don't need to have all the skills that we are listing, although that helps,
+of course. But you need to demonstrate to us that you're able and keen to learn
```

Comment:
```suggestion
The listed skills are suggested rather than essential, although they will be used
as part of our decision criteria in choosing GSoC contributors. Our only essential
requirement is that you need to demonstrate to us that you're able and keen to learn
```

Reply:
Is this fully true? I got the feeling from folks that some of these really should be non-optional skills (note, I'm happy with making them non-optional).